examples/gallery/demos/bokeh/nyc_airport_connections.ipynb

```diff
@@ -69,13 +68,20 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "plot = (graph * nyc_labels * other_labels).relabel('NYC Airport Connections')\n",
+    "nyc_labels = labels.select(City='New York').opts(\n",
+    "    text_color='white', yoffset=0.05, text_font_size='16pt')\n",
     "\n",
-    "plot.opts(\n",
-    "    opts.Graph(bgcolor='gray', edge_color=dim('SourceID').categorize({3697: 'red', 3797: 'blue'}, 'gray'), \n",
-    "               node_color=dim('index').categorize({3697: 'red', 3797: 'blue'}, 'gray'), width=800, height=800, \n",
+    "other_labels = labels[labels['City']!='New York'].opts(\n",
+    "    text_color='white', text_font_size='8pt')\n",
+    "\n",
+    "cmap = {3697: 'red', 3797: 'blue'}\n",
+    "\n",
+    "(graph * nyc_labels * other_labels).opts(\n",
+    "    opts.Graph(bgcolor='gray', width=800, height=800,\n",
+    "               edge_color=dim('SourceID').categorize(cmap, 'gray'), \n",
+    "               node_color=dim('index').categorize(cmap, 'gray'), \n",
     "               xaxis=None, yaxis=None, xlim=(-1.2, 1.2), ylim=(-1.2, 1.2))\n",
-    ")"
+    ").relabel('NYC Airport Connections')"
```

Comment:
Again, I am not sure about `relabel` after `.opts`. I can buy the argument that `.cols` is about display, but I don't think I agree with `relabel`. In this case, why not set the title as an option?

Reply:
I quite like this style, but won't object to switching to ``title``. Looking forward to that PR.